### DISCUSSION TITLE
UI Access control [Apr 02]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1634] [Discord Threads]
> 🙋‍♂️ **Harbard**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Harbard@discord Right now, roles is the only way you can control access control. Custom roles & ACL based on users is not supported right now.  
#  
> 🙋‍♂️ **Harbard**  
#### Alright, thanks !   
#### I see the create role button is "coming soon", is it in the roadmap for the next release or not at all ? 😄  
#  
> 👨‍🦲 **dstala**  
#### It is one amongst several features listed in our roadmap. We will crack them all soon. 🙂  
#  
> 🙋‍♂️ **Harbard**  
#### Alright, and would a PR be welcome on this matter or not ?  
#  
> 👨‍🦲 **dstala**  
#### Happy to accept PR!  
#  
> 🙋‍♂️ **Harbard**  
#### Alright, I guess next step will be to learn Vue and TypeScript then <:sbPsySweat:603168905917366302>  
#  
> 👨‍🦲 **dstala**  
#### 🙂  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/959128266453684284)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
